app/src/main/res/layout/opoc_filesystem_item.xml

```diff
@@ -46,6 +48,10 @@
             android:textSize="19sp"
             tools:text="how to write a readme.md" />
 
+        <Space
+            android:layout_width="8dp"
+            android:layout_height="wrap_content" />
```

Comment:
Makes default vertical layout 8dp bigger by adding this spacer. If it's needed for horiziontal, hide it in vertical please

Reply:
To add to this. 

IMO if we can come up with a slightly more compact clean layout which doesn't compromise anything we should just switch to that as the new default.